tests/integration/test_plugins.py

```diff
@@ -248,16 +250,20 @@ def test_logical_line_plugin(tmpdir, capsys):
     cfg.write(cfg_s)
 
     src = """\
-f'hello world'
+f'{{"{hello}": "{world}"}}'
```

Comment:
please write a separate test demonstrating your problem instead of changing the meaning of this test

Reply:
I meant an entirely separate test -- with a name describing what's being tested (specifically the handling of curly braces in fstrings) rather than overloading the existing test

Reply:
Seems a bit overkill, no? This accomplishes the same thing without significant additional runtime cost, and it's arguably a better test owing or the fact it mixes strings and "code". From your comment on the issue:

> nope! the method is meant to redact string contents and not code

I can straight up copy-paste 99% of the test, but it just seems a bit daft.

Reply:
I've asked for it twice now